src/templates/ingress-daemon-set.yaml

```diff
@@ -230,6 +230,21 @@ rules:
     verbs:
       - create
       - patch
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    resourceNames:
+      - ingress-controller-leader
+    verbs:
+      - get
+      - update
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - create 
```

Comment:
Fix for the CVE

Reply:
Is there any reference from upstream that required this change?

Reply:
its a backport of #180

Reply:
[upstream reference](https://github.com/kubernetes/ingress-nginx/blob/8aca34bfaa3589f9ae5ca300df4d8d825424eb28/charts/ingress-nginx/templates/controller-role.yaml#L65)